src/vue/vuedongsan/src/assets/oneroom.js

```diff
@@ -0,0 +1,42 @@
+export default [{
+    id : 0,
+    title: "Sinrim station 30 meters away",
+    image: "https://codingapple1.github.io/vue/room0.jpg",
+    content: "18년 신축공사한 남향 원룸 ☀️, 공기청정기 제공",
+    price: 340000
```

Comment:
괄호 안에 있는 JSON 요소들은 탭으로 Indent를 넣어주세요

```suggestion
      id : 0,
      title: "Sinrim station 30 meters away",
      image: "https://codingapple1.github.io/vue/room0.jpg",
      content: "18년 신축공사한 남향 원룸 ☀️, 공기청정기 제공",
      price: 340000
```

Reply:
EOF까지 추가 했습니다 :D